### DISCUSSION TITLE
Spectrum analyser code is not compatible with latest firmware

### DISCUSSION THREAD

**Discussion by lokedhs** [djsime1/awesome-flipperzero#46] [General]
I fixed the code so that it compiles with the latest framework. You can find my patch here: https://codeberg.org/loke/flipperzero-firmware/commit/bd3058d75f4d7d9e7ab39388ce99e87093048555

This runs with the latest filpperzero-firmware as of today.

**Comment by djsime1** [comment]
Is this a direct copy with the addition of the application.fam file or are there other code changes? I'm deciding if I should replace the link with this one or Unleashed's modifications.

**Reply by lokedhs** [reply]
It is a direct copy that is fixed to use the new flipperzero API's. I later noticed that this was already included in the unleashed firmware, so there isn't really a need for my port.
